examples/6.educational_pluto_notebook/simulation-example-solution.jl

```diff
@@ -277,73 +299,73 @@ The spin echo experiment has the advantage that the echo signal amplitud it is m
 
 For this section we will use the phantom `obj_t2star` and a new sequence `seq_se`.
 
-Our sequence consists of:
- - (4.1) A 90deg hard RF pulse
- - (4.2) A `Delay` of $$\mathrm{TE}/2$$ with a positive gradient (area `Ax`)
- - (4.3) A 180deg hard RF pulse
- - (4.4) A readout gradient of area `2Ax` with an ADC (similar to (2.2)), such that the middle of the gradient and ADC are in $$\mathrm{TE}$$
- - (4.5) Create concatenating these blocks into a sequence called `seq_se`
- - (4.6) Plot `seq_se` and its k-space. Is the k-space the same as `seq_gre` in (2.3)?
+- (4.1) Create a sequence called `seq_se` with the following blocks:
+  * (4.1.1) A 90deg hard RF pulse
+  * (4.1.2) A `Delay` of $$\mathrm{TE}/2$$ with a positive gradient (area `Ax`)
+  * (4.1.3) A 180deg hard RF pulse
+  * (4.1.4) A readout gradient of area `2Ax` with an ADC (similar to (2.2)), such that the middle of the gradient and ADC are in $$\mathrm{TE}$$
+ 
+ - (4.2) Plot `seq_se` and its k-space. Is the k-space the same as `seq_gre` in (2.3)?
 """
 
 # ╔═╡ 27e65680-22a0-4079-b6df-d60a3218e52e
-# (4.1) A 90deg hard RF pulse
+# (4.1) Create a Spin Echo sequence `seq_se`
 begin
+	# (4.1.1) A 90deg hard RF pulse
 	seq_se = Sequence()
 	seq_se += rf
-# (4.2) A `Delay` of TE/2 with a positive gradient (area `Ax`)
+    # (4.1.2) A `Delay` of TE/2 with a positive gradient (area `Ax`)
 	seq_se += -1*gx_pre
+	# (4.1.3) A 180deg hard RF pulse
 	seq_se += (0.0+2.0im)*rf
-# (4.3) A 180deg hard RF pulse
+	# (4.1.4) A readout gradient
 	seq_se += readout
 end
 
 # ╔═╡ f1f3b700-5916-496f-b938-46f7f08b4eb6
-# (4.6) Plot seq_se and its k-space. Is the k-space the same as seq_gre in (2.3)?
+# (4.2) Plot seq_se and its k-space. Is the k-space the same as seq_gre in (2.3)?
 plot_seq(seq_se; slider=false)
 
 # ╔═╡ 4e1434e1-673f-4206-a271-9edec10ebd6a
 plot_kspace(seq_se)
 
 # ╔═╡ 45952512-aaf1-43d8-a95e-c32bb2633f42
 md"""
- - (4.7) Simulate using `seq_se` and `obj_t2star`
- - (4.8) Compare the signal obtained in (4.6) with the one at (3.5)
- - (4.9) Reconstruct the 1D image
+ - (4.3) Simulate using `seq_se` and `obj_t2star`
+ - (4.4) Compare the signal obtained in (4.6) with the one at (3.5)
```

Comment:
This references are wrong now

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/253/commits/3fde402ca7a7613045f15f2c77564a4f5491fcfb